lib/sdf-server/Cargo.toml

```diff
@@ -9,7 +9,6 @@ rust-version.workspace = true
 publish.workspace = true
 
 [dependencies]
-async-openai = { workspace = true }
```

Comment:
I noticed we still had this in here, when we don't use it--so I removed it. Cargo-only change.